config.toml

```diff
@@ -14,3 +14,7 @@ OPENAI = "<YOUR_OPENAI_API_KEY>"
 
 [API_ENDPOINTS]
 BING = "https://api.bing.microsoft.com/v7.0/search"
+OPENAI_BASE_URL = "https://api.openai.com/v1"
```

Comment:
**Suggestion**: Consider adding validation or a default value for the "SEARCH_ENGINE" preference to ensure it only accepts "Bing" or "DDGS" as valid inputs. This validation could prevent potential runtime errors due to misconfiguration. <br> **Code Suggestion**: 
 ```
 +SEARCH_ENGINE = "Bing" # Possible Values: Bing, DDGS. Ensure to validate this field to accept only these values.
 ```